src/experimental-features.js

```diff
@@ -0,0 +1,26 @@
+export class ExperimentalFeatures {
```

Comment:
What do you think about adding it to our [composition root](https://github.com/FromDoppler/doppler-webapp/blob/master/src/services/pure-di.tsx#L44) ?

Reply:
Yes, I thought that. I will do so.